docs/mini-apps/features/context.mdx

```diff
@@ -0,0 +1,390 @@
+---
+title: "Understanding Mini App Context"
+description: "Leverage context data to build social-first mini applications that adapt to user entry points and drive viral growth"
+sidebarTitle: "Context"
+---
+
+Context is the environmental data that describes how, where, and by whom your mini app was launched. This guide shows you how to leverage context to create personalized experiences and optimize viral growth.
```

Comment:
suggestion: "Context is data that is automatically provided to your mini app which describes how, where, and..."

I think specifying that it's automatically available is a nice detail to put up-front.